### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by steeling**

### CODE REVIEW

File: pkg/dss/server.go [hunk 1]
```diff
@@ -52,65 +29,14 @@ type Store interface {
 	SearchSubscriptions(ctx context.Context, cells s2.CellUnion, owner string) ([]*dspb.Subscription, error)
 }
 
-type loggingStore struct {
```
Comment (vosst) [inline pkg/dss/server.go]: Hmmm, I would appreciate if we could leave a logging decorator available for debugging purposes. While I see your changes to introduce an interceptor, we would loose structured logs, and everything becomes a string.

What do you think?

File: pkg/dss/geo/s2.go [hunk 2]
```diff
@@ -57,36 +55,44 @@ func splitAtComma(data []byte, atEOF bool) (int, []byte, error) {
...
 		scanner  = bufio.NewScanner(strings.NewReader(area))
 	)
+	numCoords := strings.Count(area, ",")/2 + 1
+	if numCoords%2 == 1 {
+		return nil, errOddNumberOfCoordinatesInAreaString
+	}
+	if numCoords < 3 {
+		return nil, errNotEnoughPointsInPolygon
+	}
+	fmt.Println(numCoords)
```
Comment (vosst) [inline pkg/dss/geo/s2.go]: This seems to be a left-over.

File: pkg/dss/geo/s2.go [hunk 3]
```diff
@@ -18,20 +19,17 @@ const (
...
-	// DefaultRegionCoverer is the default s2.RegionCoverer for mapping areas
+	// defaultRegionCoverer is the default s2.RegionCoverer for mapping areas
 	// and extents to s2.CellUnion instances.
-	DefaultRegionCoverer = &s2.RegionCoverer{
+	defaultRegionCoverer = &s2.RegionCoverer{
 		MinLevel: DefaultMinimumCellLevel,
 		MaxLevel: DefaultMaximumCellLevel,
 	}
+	// RegionCoverer provides an overridable interface to defaultRegionCoverer
+	RegionCoverer = defaultRegionCoverer
```
Comment (vosst) [inline pkg/dss/geo/s2.go]: Nice!

File: pkg/dss/server.go (hunk 1 above)
Comment (steeling) [reply pkg/dss/server.go]: Ah, I thought with CodeGenRequestFieldExtractor we could preserve some of the structured logging, but I never verified this. re-added

**Pull request merged**